.babelrc

```diff
@@ -10,6 +10,7 @@
     },
     "production": {
       "plugins": [
+        "transform-react-remove-prop-types",
```

Comment:
Ah, now I get it. 😆 You assume that I'm using Node. ;-) But you wouldn't use the bundle there, would you?
Actually, I'm talking about using the UMD build in a browser (`<script src=...`) or in an AMD environment (RequireJS).